### PR TITLE
Update ufs-weather for latest CICE emc/develop; Update to esmf-bs47; add export of openwater_frac_in_atm from fv3ATM when coupled

### PR DESCRIPTION
**Pull request by DeniseWorthen**
## Description

This PR will: 

* update CICE in ufs-weather for the latest emc/develop at NOAA-EMC/CICE.
* update fv3ATM to use esmf-bs47, add the export of open-water fraction from fv3ATM and update module files across all platforms to use bs47 (PR #493)

### Issue(s) addressed

- CICE Issue [#25](https://github.com/NOAA-EMC/CICE/issues/25)
- UFS Issue #494 
- partial fix to issue UFS #366. FV3atm will export the required field but no use will be made of it until changes are made in the CMEPS mediator.

## Testing

* This PR will change all ufs-coupled and ufs-datm baselines due to a non-b4b bugfix in Icepack. A test baseline was created on hera.intel and all cpld and datm regression tests passed against that test baseline.

* This PR also changes baselines due to the change to esmf-bs47 because some forecast files will contain different metadata or different ordering of fields. See fv3ATM Issue [#265](https://github.com/NOAA-EMC/fv3atm/issues/265) for more details on the changes expected.

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3

## Dependencies
NEMS [PR#94](https://github.com/NOAA-EMC/NEMS/pull/94)
CICE [PR #24](https://github.com/NOAA-EMC/CICE/pull/24)
CMEPS [PR #39](https://github.com/NOAA-EMC/CMEPS/pull/39)
FV3-ATM [PR #266](https://github.com/NOAA-EMC/fv3atm/pull/266)

### CODE REVIEW

**Comment by MinsukJi-NOAA** [comment]
@DeniseWorthen , this fixes the compare_ncfile issue on Cray:
```
diff --git a/tests/rt.sh b/tests/rt.sh
index b342a2b..d9b1566 100755
--- a/tests/rt.sh
+++ b/tests/rt.sh
@@ -120,7 +120,6 @@ source $PATHTR/NEMS/src/conf/module-setup.sh.inc
 if [[ $MACHINE_ID = wcoss_cray ]]; then
 
   module load xt-lsfhpc
-  module load python/3.6.3
 
   module use /usrx/local/emc_rocoto/modulefiles
   module load rocoto/1.3.0rc2
@@ -129,10 +128,10 @@ if [[ $MACHINE_ID = wcoss_cray ]]; then
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=lsfcray
 
-  module use /gpfs/hps/nco/ops/nwtest/modulefiles
   module load ecflow/intel/4.17.0.1
   ECFLOW_START=${ECF_ROOT}/bin/ecflow_start.sh
   ECF_PORT=$(grep $USER /usrx/local/sys/ecflow/assigned_ports.txt | awk '{print $2}')
+  module load python/3.6.3
 
   DISKNM=/gpfs/hps3/emc/nems/noscrub/emc.nemspara/RT
   QUEUE=debug
```
Test result is here: `/gpfs/hps3/emc/nems/noscrub/Minsuk.Ji/PR475/tests/RegressionTests_wcoss_cray.log`

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA 

I made these changes:
```
diff --git a/tests/rt.sh b/tests/rt.sh
index b342a2b..d9b1566 100755
--- a/tests/rt.sh
+++ b/tests/rt.sh
@@ -120,7 +120,6 @@ source $PATHTR/NEMS/src/conf/module-setup.sh.inc
 if [[ $MACHINE_ID = wcoss_cray ]]; then

   module load xt-lsfhpc
-  module load python/3.6.3

   module use /usrx/local/emc_rocoto/modulefiles
   module load rocoto/1.3.0rc2
@@ -129,10 +128,10 @@ if [[ $MACHINE_ID = wcoss_cray ]]; then
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=lsfcray

-  module use /gpfs/hps/nco/ops/nwtest/modulefiles
   module load ecflow/intel/4.17.0.1
   ECFLOW_START=${ECF_ROOT}/bin/ecflow_start.sh
   ECF_PORT=$(grep $USER /usrx/local/sys/ecflow/assigned_ports.txt | awk '{print $2}')
+  module load python/3.6.3

   DISKNM=/gpfs/hps3/emc/nems/noscrub/emc.nemspara/RT
   QUEUE=debug
```

But when I try ``./rt.sh -e >output 2>&1 &`` I get in output:

```
++ /opt/modules/3.2.10.3/bin/modulecmd bash load ecflow/intel/4.17.0.1
ecflow/intel(3):ERROR:105: Unable to locate a modulefile for 'ecflow/intel/4.17.0.1'
+ eval
./rt.sh: line 132: ECF_ROOT: unbound variable
```

**Comment by MinsukJi-NOAA** [comment]
@DeniseWorthen can you put this line back? `module use /gpfs/hps/nco/ops/nwtest/modulefiles`

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA That solved the ECF_ROOT problem. I'll go ahead and let the tests run and see how it goes. I can easily make the change manually if everything checks out. Thanks.

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA The cray jobs completed with the expected results: regional_quilt tests fail (GRBF00 files differ), and the hwrf_thompson tests fail because of the dyn/phyf 000 forecast files differ. The ALT CHECK worked correctly in all other cases. I will put this fix in the merged PR branch. Thanks.

**Comment by MinsukJi-NOAA** [comment]
> @MinsukJi-NOAA The cray jobs completed with the expected results: regional_quilt tests fail (GRBF00 files differ), and the hwrf_thompson tests fail because of the dyn/phyf 000 forecast files differ. The ALT CHECK worked correctly in all other cases. I will put this fix in the merged PR branch. Thanks.

Thanks for testing @DeniseWorthen

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA can you assign reviewers for the new NEMS PR [94](https://github.com/NOAA-EMC/NEMS/pull/94) to update the field dictionary. Thanks.

**Comment by MinsukJi-NOAA** [comment]
> @MinsukJi-NOAA can you assign reviewers for the new NEMS PR [94](https://github.com/NOAA-EMC/NEMS/pull/94) to update the field dictionary. Thanks.

Done.

**Comment by BrianCurtis-NOAA** [comment]
Machine: hera
Compiler: intel
Job: RT
Repo location: /scratch1/NCEPDEV/nems/emc.nemspara/autort/pr/592556664/20210401143042/ufs-weather-model
Please manually delete: /scratch1/NCEPDEV/stmp2/emc.nemspara/FV3_RT/rt_156419
Test 026 fv3_regional_quilt FAIL
Please make changes and add the following label back:
hera-intel-RT

**Comment by DeniseWorthen** [comment]
The regional_quilt failed comparisons on the NATLEV.GrbF00 and NATLEV.GrbF24. I don't know anything about these files or why they would have not been reproducible. 

When testing the esmfbs47 they did come up as different than the then-current baseline.

**Comment by BrianCurtis-NOAA** [comment]
> The regional_quilt failed comparisons on the NATLEV.GrbF00 and NATLEV.GrbF24. I don't know anything about these files or why they would have not been reproducible.
> 
> When testing the esmfbs47 they did come up as different than the then-current baseline.

So do we need new baselines?

**Comment by junwang-noaa** [comment]
Those are two post files, I guess if the data type of ESMF attributes is
changed,  and the post reads them, it may impact the post results slightly.

**Comment by DeniseWorthen** [comment]
They're comparing against the new baseline so there must be some variation in the files like Jun said. Should I remove the comparison of the grb files for this test for now? There are 4 of them PRSLEV.GrbF00,24 and NATLEV.Grb00,24.

**Comment by climbfuji** [comment]
If they change from run to run using the same code, then that is a red flag for potentially uninitialized variables etc - run the test in full debug mode?

**Comment by DeniseWorthen** [comment]
OK, I'll give that a try.

**Comment by DeniseWorthen** [comment]
I compiled fv3_regional_quilt with DEBUG=Y (``COMPILE | APP=ATM SUITES=FV3_GFS_v15_thompson_mynn 32BIT=Y DEBUG=Y``); the run failed with a FPE here

/scratch1/NCEPDEV/stmp2/Denise.Worthen/FV3_RT/rt_212108/fv3_regional_quilt

**Comment by junwang-noaa** [comment]
Will need some time to debug this test with ESMF810bs47. Does this happen
on all the platforms?

**Comment by DeniseWorthen** [comment]
I will try the same on debug test on cray.

**Comment by DeniseWorthen** [comment]
Putting the regional_quilt test in debug mode on cray resulted in the same error:
```
forrtl: error (65): floating invalid
Image              PC                Routine            Line        Source
fv3.exe            0000000009F31ECE  Unknown               Unknown  Unknown
fv3.exe            000000000A7E7F60  Unknown               Unknown  Unknown
fv3.exe            00000000033F21C3  post_regional_mp_        1375  post_regional.F90
```

**Comment by DusanJovic-NOAA** [comment]
Line 1375 in post_regional.F90 is:

`if (abs(arrayr42d(i,j)-fillValue) < small) islope(i,j) = spval`

`islope` is integer, `spval` is real. Instead of assigning real to integer (which might overflow) try:

`if (abs(arrayr42d(i,j)-fillValue) < small) islope(i,j) = huge(1)`

**Comment by DeniseWorthen** [comment]
I will try Dusan's fix. If I it runs, I'll try creating a non-debug baseline and verify against it.

**Comment by DeniseWorthen** [comment]
The fix did not work. I'm not sure where the error is now, since I can't find the listed routines:

```
 in fcst run phase 2, na=           0
forrtl: error (72): floating overflow
Image              PC                Routine            Line        Source
fv3.exe            0000000009F31ECE  Unknown               Unknown  Unknown
fv3.exe            000000000A7E7F60  Unknown               Unknown  Unknown
fv3.exe            000000000A3FAC2B  caldwp_                    65  CALDWP.f
fv3.exe            000000000A33D322  mdlfld_                  1577  MDLFLD.f
fv3.exe            000000000A13EC5D  process_                   81  PROCESS.f
fv3.exe            0000000003340859  post_regional_mp_         201  post_regional.F90
fv3.exe            000000000322E5C5  inline_post_mp_in          49  inline_post.F90
```

**Comment by DusanJovic-NOAA** [comment]
It worked, code passed that point where it was previously crashing. This is now different error, in the post library itself. I guess it's because the computation in CALDWP.f is not checking for undefined points outside the model domain.

**Comment by DusanJovic-NOAA** [comment]
Try running without DEBUG=Y, and see if you get reproducible outputs between two consecutive runs.

**Comment by DeniseWorthen** [comment]
Ok, will do.

**Comment by DeniseWorthen** [comment]
On dell, three baseline tests have failed to create (twice) because of running out of wall time:

```
fv3_regional_control
fv3_gfdlmprad
fv3_gfdlmprad_atmwav
```

The gfdlmprad test is only getting to hour 18 but it looks like it is supposed to run to 24. I'm going to try setting WLCLK to 1 hour for all three.

**Comment by DeniseWorthen** [comment]
So, for the regional_quilt test, I built it w/o debug mode, created the single baseline and tried to run against it. Previously, the NATLEV.GrbF00 and NATLEV.GrbF24 did not reproduce. This time, the PRSLEV.GrbF00 and PRSLEV.GrbF24 did not reproduce: 

```
baseline dir = /gpfs/hps3/stmp/Denise.Worthen/FV3_RT/test_baseline/fv3_regional_quilt
working dir  = /gpfs/hps3/stmp/Denise.Worthen/FV3_RT/rt_22219/fv3_regional_quilt
Checking test 001 fv3_regional_quilt results ....
 Comparing dynf000.nc .........OK
 Comparing dynf024.nc .........OK
 Comparing phyf000.nc .........OK
 Comparing phyf024.nc .........OK
 Comparing PRSLEV.GrbF00 .........NOT OK
 Comparing PRSLEV.GrbF24 .........NOT OK
 Comparing NATLEV.GrbF00 .........OK
 Comparing NATLEV.GrbF24 .........OK
```

**Comment by junwang-noaa** [comment]
@DeniseWorthen I'd suggest creating an issue and then comment out this test in RT. It may need some time to figure out what causes the run-to-run differences in POST files, it's puzzled to me why we do not have the problem before.

**Comment by DeniseWorthen** [comment]
OK, will do. That means I have baselines now on hera gnu/intel, cheyenne gnu/intel, dell-p3 and cray. The hera.intel should verify once we remove the regional_quilt test. 

I'll start the verify runs on the wcoss machines and cheyenne. I'll wait for those to complete before starting the baseline creation on orion,jet and gaea.

The three tests on dell-p3 took ~20mins

**Comment by MinsukJi-NOAA** [comment]
@DeniseWorthen CI test failure is likely due to ESMF library change, which also needs to be implemented in the Docker container.

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA So will we need to skip the ci tests for this commit, or is updating the container something quick?

**Comment by MinsukJi-NOAA** [comment]
> @MinsukJi-NOAA So will we need to skip the ci tests for this commit, or is updating the container something quick?

It will take time since it needs to be tested.

**Comment by DeniseWorthen** [comment]
@junwang-noaa all the submodules have approvals at this point except CICE--can you please review/approve?

**Comment by DeniseWorthen** [comment]
I put the jet baseline in the wrong location for the auto-rt to find it (``/lfs4/HFIP/hfv3gfs/RT/NEMSfv3gfs`` vs ``/lfs4/HFIP/hfv3gfs/emc.nemspara/RT/NEMSfv3gfs``) so we'll be restarting the jet auto-rt.

**Comment by BrianCurtis-NOAA** [comment]
Machine: jet
Compiler: intel
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/592556664/20210402133014/ufs-weather-model
Please manually delete: /lfs4/HFIP/hfv3gfs/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_192782
Test fv3_multigases 016 failed in check_result failed 
Test fv3_multigases 016 failed in run_test failed 
Test fv3_control 001 failed in check_result failed 
Test fv3_control 001 failed in run_test failed 
Test fv3_lheatstrg 015 failed in check_result failed 
Test fv3_wrtGauss_netcdf_parallel 007 failed in check_result failed 
Test fv3_lheatstrg 015 failed in run_test failed 
Test fv3_wrtGauss_netcdf_parallel 007 failed in run_test failed 
Test fv3_control_32bit 017 failed in check_result failed 
Test fv3_control_32bit 017 failed in run_test failed 
Test fv3_lndp 013 failed in check_result failed 
Test fv3_lndp 013 failed in run_test failed 
Test fv3_2threads 002 failed in check_result failed 
Test fv3_2threads 002 failed in run_test failed 
Test fv3_wrtGlatlon_netcdf 008 failed in check_result failed 
Test fv3_wrtGlatlon_netcdf 008 failed in run_test failed 
Test fv3_wrtGauss_netcdf 006 failed in check_result failed 
Test fv3_wrtGauss_netcdf 006 failed in run_test failed 
Test fv3_stochy 011 failed in check_result failed 
Test fv3_stochy 011 failed in run_test failed 
Test fv3_wrtGauss_nemsio 009 failed in check_result failed 
Test fv3_wrtGauss_nemsio 009 failed in run_test failed 
Test fv3_gfdlmp 024 failed in check_result failed 
Test fv3_gfdlmp 024 failed in run_test failed 
Test fv3_wrtGauss_nemsio_c192 010 failed in check_result failed 
Test fv3_wrtGauss_nemsio_c192 010 failed in run_test failed 
Test fv3_ca 012 failed in check_result failed 
Test fv3_ca 012 failed in run_test failed 
Test fv3_gfdlmprad_noahmp 026 failed in check_result failed 
Test fv3_gfdlmprad_noahmp 026 failed in run_test failed 
Test fv3_gfdlmprad_gwd 025 failed in check_result failed 
Test fv3_gfdlmprad_gwd 025 failed in run_test failed 
Test fv3_wrtGauss_netcdf_esmf 005 failed in check_result failed 
Test fv3_wrtGauss_netcdf_esmf 005 failed in run_test failed 
Test fv3_stretched 018 failed in check_result failed 
Test fv3_stretched 018 failed in run_test failed 
Test fv3_stretched_nest 019 failed in check_result failed 
Test fv3_stretched_nest 019 failed in run_test failed 
Test fv3_regional_control 020 failed in check_result failed 
Test fv3_regional_control 020 failed in run_test failed 
Test fv3_regional_quilt_netcdf_parallel 023 failed in check_result failed 
Test fv3_regional_quilt_netcdf_parallel 023 failed in run_test failed 
Test fv3_regional_quilt_hafs 022 failed in check_result failed 
Test fv3_regional_quilt_hafs 022 failed in run_test failed 
Test fv3_gfs_v16_flake 045 failed in check_result failed 
Test fv3_gfs_v16_flake 045 failed in run_test failed 
Test fv3_gocart_clm 044 failed in check_result failed 
Test fv3_gocart_clm 044 failed in run_test failed 
Test fv3_gfsv16_csawmgt 043 failed in check_result failed 
Test fv3_gfsv16_csawmgt 043 failed in run_test failed 
Test fv3_satmedmfq 029 failed in check_result failed 
Test fv3_satmedmfq 029 failed in run_test failed 
Test fv3_satmedmf 028 failed in check_result failed 
Test fv3_satmedmf 028 failed in run_test failed 
Test fv3_csawmg 027 failed in check_result failed 
Test fv3_csawmg 027 failed in run_test failed 
Test fv3_gfsv16_csawmg 042 failed in check_result failed 
Test fv3_gfsv16_csawmg 042 failed in run_test failed 
Test fv3_gfdlmp_32bit 030 failed in check_result failed 
Test fv3_gfdlmp_32bit 030 failed in run_test failed 
Test fv3_gfdlmprad_32bit_post 031 failed in check_result failed 
Test fv3_gfdlmprad_32bit_post 031 failed in run_test failed 
Test fv3_thompson 034 failed in check_result failed 
Test fv3_thompson 034 failed in run_test failed 
Test fv3_thompson_no_aero 035 failed in check_result failed 
Test fv3_thompson_no_aero 035 failed in run_test failed 
Test fv3_gsd 033 failed in check_result failed 
Test fv3_gsd 033 failed in run_test failed 
Test fv3_control_debug 054 failed in check_result failed 
Test fv3_control_debug 054 failed in run_test failed 
Test fv3_cpt 032 failed in check_result failed 
Test fv3_cpt 032 failed in run_test failed 
Test fv3_rrfs_v1beta_debug 060 failed in check_result failed 
Test fv3_rrfs_v1beta_debug 060 failed in run_test failed 
Test fv3_gsd_debug 056 failed in check_result failed 
Test fv3_gsd_debug 056 failed in run_test failed 
Test fv3_gsd_diag3d_debug 057 failed in check_result failed 
Test fv3_gsd_diag3d_debug 057 failed in run_test failed 
Test fv3_thompson_no_aero_debug 059 failed in check_result failed 
Test fv3_thompson_no_aero_debug 059 failed in run_test failed 
Test fv3_gfs_v16_debug 051 failed in check_result failed 
Test fv3_gfs_v16_debug 051 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 061 failed in check_result failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 061 failed in run_test failed 
Test fv3_thompson_debug 058 failed in check_result failed 
Test fv3_thompson_debug 058 failed in run_test failed 
Test fv3_stretched_nest_debug 055 failed in check_result failed 
Test fv3_stretched_nest_debug 055 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_debug 052 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP_debug 052 failed in run_test failed 
Test fv3_gfs_v16_ras_debug 064 failed in check_result failed 
Test fv3_gfs_v16_ras_debug 064 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 062 failed in check_result failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 062 failed in run_test failed 
Test fv3_gfsv16_ugwpv1_debug 063 failed in check_result failed 
Test fv3_gfsv16_ugwpv1_debug 063 failed in run_test failed 
Test fv3_regional_control_debug 053 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_2thrd 041 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP_2thrd 041 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_c192L127 040 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP_c192L127 040 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP 039 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP 039 failed in run_test failed 
Test fv3_gfs_v16_stochy 038 failed in run_test failed 
Test fv3_gfs_v16_restart 037 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 046 failed in check_result failed 
Test fv3_HAFS_v0_hwrf_thompson 046 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 047 failed in run_test failed 
Test fv3_gfs_v16_ras 050 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-RT

**Comment by BrianCurtis-NOAA** [comment]
Machine: jet
Compiler: intel
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/592556664/20210402160009/ufs-weather-model
Please manually delete: /lfs4/HFIP/hfv3gfs/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_18641
Test fv3_stretched_nest 019 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 046 failed in check_result failed 
Test fv3_HAFS_v0_hwrf_thompson 046 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-RT

**Comment by DeniseWorthen** [comment]
The fv3_stretched_nest failed due to time wall clock but it actually seemed to have hung. It is a 48hr forecast and it looks like it only got a few timesteps. 

The HAFS test failed on the 4x daily tile2 because the baseline file has no time level information written: 

```
time = UNLIMITED ; // (0 currently)
```

The other 4x daily files for the other tiles have 4 timesteps correctly written and they compare identically with the verify run.

I will add that I'm having some trouble getting response from jet when I ``ls or cd``. So perhaps there is a system issue going on here too.

**Comment by DeniseWorthen** [comment]
The baselines on jet are not completing (compiling) because the disk is full:

CMakeFiles/ccppphys.dir/physics/GFS_phys_time_vary.fv3.F90.o: catastrophic error: Disk is full, no space to write object file

This is in my own attempt to run the last two baselines as well as the auto-rt I started late yesterday.

**Comment by BrianCurtis-NOAA** [comment]
Machine: jet
Compiler: intel
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/592556664/20210402230015/ufs-weather-model
Please manually delete: /lfs4/HFIP/hfv3gfs/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_150647
Test compile 014 failed failed 
Test compile 013 failed failed 
Test fv3_cpt 032 failed in run_test failed 
Test fv3_wrtGauss_nemsio_c192 010 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_2thrd 041 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 047 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-RT

**Comment by DeniseWorthen** [comment]
Ready for review & merge.

File: tests/rt.conf
Comment (climbfuji) [inline tests/rt.conf:43]: Do we have a timeline for fixing this?

File: tests/tests/fv3_gfdlmprad
Comment (climbfuji) [inline tests/tests/fv3_gfdlmprad:59]: These longer runtimes need to be investigated.

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]

**Review by BrianCurtis-NOAA — APPROVED** [review]

File: tests/rt.conf
Comment (DeniseWorthen) [reply tests/rt.conf:43]: I think Jun has tracked down the issues [here](https://github.com/ufs-community/ufs-weather-model/issues/504). She's opened an associated issue in EMC_post.

File: tests/tests/fv3_gfdlmprad
Comment (DeniseWorthen) [reply tests/tests/fv3_gfdlmprad:59]: A full 60 minutes is not required but this is what I tested out of frustration. The runs ended up taking about 20-25 minutes in my case.

**Comment by MinsukJi-NOAA** [comment]
@DeniseWorthen could you include these changes? It fixes the esmf bs47 docker container issue.
```
diff --git a/tests/ci/Dockerfile b/tests/ci/Dockerfile
index c685eaf..8d01299 100644
--- a/tests/ci/Dockerfile
+++ b/tests/ci/Dockerfile
@@ -1,4 +1,4 @@
-From noaaemc/ubuntu-hpc:v1.1
+From noaaemc/ubuntu-hpc:v1.2
 
 ENV HOME=/home/builder
 COPY --chown=builder:builder . $HOME/ufs-weather-model
```

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA Should I now try run-ci?

**Comment by MinsukJi-NOAA** [comment]
Yes, that would be great.

**Comment by DeniseWorthen** [comment]
@MinsukJi-NOAA Two CI tests failed. Should I hold off merging until you can take a look?

**Comment by MinsukJi-NOAA** [comment]
I will take a look. I suggest you go ahead with merging for this PR.

**Pull request merged**
